### PR TITLE
Replace - with _ when generating python modules

### PR DESCRIPTION
**Pull request by notmgsk**

### CODE REVIEW

File: update_python_spec.lisp [hunk 1]
```diff
@@ -1,13 +1,23 @@
 #!/usr/bin/env sbcl --script
 (load "~/.sbclrc")
 (ql:quickload :rpcq)
+(ql:quickload :split-sequence)
 (rpcq::clear-messages)
 
+(defun -to_ (f)
```
Comment (notmgsk) [inline update_python_spec.lisp]: Maybe I'm tired and that's why I can't write a simple solution for search and replace, or find one within CL.

Comment (ecpeterson) [reply update_python_spec.lisp]: We have a variant of this function elsewhere in rpcq already: https://github.com/rigetti/rpcq/blob/master/src/utilities.lisp#L8 .

Comment (stylewarning) [reply update_python_spec.lisp]: @notmgsk 
```
CL-USER> (substitute #\_ #\- "Hello-world_there")
"Hello_world_there"
```

File: update_python_spec.lisp [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
 (dolist (namespace '("messages" "core-messages"))
   (load (make-pathname :name namespace :type "lisp" :directory '(:relative "src")))
-  (with-open-file (f (make-pathname :name namespace :type "py" :directory '(:relative "rpcq"))
+  (with-open-file (f (make-pathname :name (-to_ namespace) :type "py" :directory '(:relative "rpcq"))
                      :direction ':output
                      :if-exists ':supersede
                      :if-does-not-exist ':create)
     (rpcq::python-message-spec f (gethash namespace rpcq::*messages*))
-    (format t "Wrote new ~A.py~%" namespace)))
+    (format t "Wrote new ~A.py~%" (pathname-name f))))
```
Comment (stylewarning) [inline update_python_spec.lisp]: can we do something like this:
```suggestion
    (format t "Wrote new ~S~%" f)))
```

Comment (notmgsk) [reply update_python_spec.lisp]: @stylewarning `Wrote new #<SB-SYS:FD-STREAM for "file /Users/mark.skilbeck/hackery/lisp/rpcq/rpcq/messages.py" {10035492A3}>.py`

Comment (stylewarning) [reply update_python_spec.lisp]: whoa no bueno. But if that's a stream, then why are we calling `pathname-name` on it?

File: update_python_spec.lisp [hunk 3]
```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env sbcl --script
 (load "~/.sbclrc")
 (ql:quickload :rpcq)
+(ql:quickload :split-sequence)
 (rpcq::clear-messages)
 
 (dolist (namespace '("messages" "core-messages"))
   (load (make-pathname :name namespace :type "lisp" :directory '(:relative "src")))
-  (with-open-file (f (make-pathname :name namespace :type "py" :directory '(:relative "rpcq"))
```
Comment (stylewarning) [inline update_python_spec.lisp]: let's 
```
(let ((path (make-pathname ...)))
  (with-open-file (f path)
    ....
    (format t "...~S" path) 
...
```

File: update_python_spec.lisp (hunk 2 above)
Comment (notmgsk) [reply update_python_spec.lisp]: because it's a pathname

Comment (notmgsk) [reply update_python_spec.lisp]: your comment below resolves this

File: update_python_spec.lisp [hunk 4]
```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env sbcl --script
 (load "~/.sbclrc")
 (ql:quickload :rpcq)
+(ql:quickload :split-sequence)
```
Comment (ecpeterson) [inline update_python_spec.lisp]: It doesn't really matter, but do you still want this?

Comment (notmgsk) [reply update_python_spec.lisp]: removed

**Pull request merged**